packages/react-api/src/util/keyringFormat.ts

```diff
@@ -0,0 +1,14 @@
+import { KeyringPair$Json } from '@polkadot/keyring/types';
+
+export function updateToV3KeyringFormat(json: KeyringPair$Json) {
```

Comment:
I would suggest you define a type with the v2 format and change this function to get a v2 format as input and return a v3 format as output.

Reply:
done